editor.planx.uk/src/@planx/components/List/Public/index.tsx

```diff
@@ -115,39 +124,58 @@ const InactiveListCard: React.FC<{
   const { schema, formik, removeItem, editItem, isPageComponent } =
     useListContext();
 
+  const mapPreview = schema.fields.find(
+    (field) => field.data.fn === "features",
+  );
```

Comment:
nit: we decided to make the `fn` user-configurable, so I think we'll want this to be based on `field.type === "map"` rather than relying on `fn = "features"`

Reply:
Thanks for spotting, have updated 👍